_layouts/page.html

```diff
@@ -15,6 +15,11 @@
   {% if page.author.gplus %}<link rel="author" href="https://plus.google.com/{{ page.author.gplus }}">{% endif %}
   <meta property="og:title" content="{% if page.title %}{{ page.title }} {% else %}{{ site.inc.title }}{% endif %}"/>
   <meta property="og:url" content="{{ site.url }}{{ page.url }}"/>
+  {% if page.image %}
+  <meta content="/images/site-logo.svg" property="og:image">
```

Comment:
hmmm - my idea was that the image in any previews is always the upday logo - in this way no one has to think about it. But I am fine with both